packs/github/actions/lib/base.py

```diff
@@ -8,19 +8,20 @@
     'BaseGithubAction'
 ]
 
-BASE_URL = 'https://github.com'
```

Comment:
The name here is a bit misleading. Don't remove it, but rather name the variable `WEB_URL` instead. It's used for parsing data from GitHub web pages as opposed to API (that's why it's set to github.com and not api.github.com here).